.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -23,7 +23,7 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to help Modulus and fill out this feature request!
+        Thanks for taking the time to help PhysicsNemo and fill out this feature request!
```

Comment:
Maybe we should change `Earth2Studio/PhysicsNemo`

Looks like I never changed it originally :)